packages/indexer-agent/src/agent.ts

```diff
@@ -256,6 +259,20 @@ class Agent {
     )
     this.logger.info(`Waiting for network data before reconciling every 120s`)
 
+    const disputableAllocations = join({
+      currentEpoch,
+      activeDeployments,
```

Comment:
This means we are checking disputable allocations only using the indexer active subgraph deployments, is that right?

Reply:
correct, the indexer will only be able to generate reference POIs for subgraphs they are actively indexing.